frontend/src/component/changeRequest/ChangeRequest/Changes/Change/SegmentChange.tsx

```diff
@@ -20,61 +21,67 @@ export const SegmentChange: FC<ISegmentChangeProps> = ({
-                    `${theme.shape.borderRadiusLarge}px ${theme.shape.borderRadiusLarge}px 0 0`,
-                border: '1px solid',
-                borderColor: (theme) =>
-                    segmentChange.conflict
-                        ? theme.palette.warning.border
-                        : theme.palette.divider,
-                borderBottom: 'none',
+                marginTop: theme.spacing(2),
+                marginBottom: theme.spacing(2),
                 overflow: 'hidden',
             })}
         >
-            <ConflictWarning conflict={segmentChange.conflict} />
             <Box
-                sx={{
-                    display: 'flex',
-                    pt: 2,
-                    pb: 2,
-                    px: 3,
-                }}
+                sx={(theme) => ({
+                    backgroundColor: theme.palette.neutral.light,
+                    borderRadius: (theme) =>
+                        `${theme.shape.borderRadiusLarge}px ${theme.shape.borderRadiusLarge}px 0 0`,
+                    border: '1px solid',
+                    borderColor: (theme) =>
+                        segmentChange.conflict
+                            ? theme.palette.warning.border
+                            : theme.palette.divider,
+                    borderBottom: 'none',
+                    overflow: 'hidden',
+                })}
             >
-                <Typography>Segment name: </Typography>
-
-                <Link
-                    component={RouterLink}
-                    to={`/segments/edit/${segmentChange.payload.id}`}
-                    color='primary'
-                    underline='hover'
+                <ConflictWarning conflict={segmentChange.conflict} />
+                <Box
                     sx={{
-                        marginLeft: 1,
-                        '& :hover': {
-                            textDecoration: 'underline',
-                        },
+                        display: 'flex',
+                        pt: 2,
+                        pb: 2,
+                        px: 3,
                     }}
-                    onClick={onNavigate}
                 >
-                    <strong>{segmentChange.payload.name}</strong>
-                </Link>
+                    <Typography>Segment name:</Typography>
+
+                    <Link
+                        component={RouterLink}
+                        to={`/segments/edit/${segmentChange.payload.id}`}
+                        color='primary'
+                        underline='hover'
+                        sx={{
+                            marginLeft: 1,
+                            '& :hover': {
+                                textDecoration: 'underline',
+                            },
+                        }}
+                        onClick={onNavigate}
+                    >
+                        <strong>
+                            {segmentChange.payload.name || segment?.name}
```

Comment:
In deleted segments, the name was empty, so you'd end up with an empty "segment name: "
<img width="1002" alt="image" src="https://github.com/user-attachments/assets/4b52b1f0-2ec1-4248-801e-07ea26dc380f" />

Now, instead you get the segment name:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/7ef50736-432d-4cb8-83eb-b9ce79987667" />

Yes, it's repeated, but I think that's better than an empty "name".